inference-engine/src/mkldnn_plugin/emitters/jit_mkldnn_emitters.cpp

```diff
@@ -67,5 +80,24 @@ void jit_mkldnn_emitter::emit_table() {
     }
 }
 
+jit_mkldnn_aux_emitter::jit_mkldnn_aux_emitter(jit_generator *host, cpu_isa_t host_isa, const MKLDNNNode* node, InferenceEngine::Precision exec_prc)
```

Comment:
Didn't get why do we need it. Why cannot we use `jit_mkldnn_emitter`?

Reply:
discussed offline